Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py

```diff
@@ -106,12 +110,31 @@ async def get_recent_trades_history(self, exchange, symbol):
     async def get_ohlcv_history(self, exchange, symbol, time_frame):
         # use time_frame_sec to add time to save the candle closing time
         time_frame_sec = commons_enums.TimeFramesMinutes[time_frame] * commons_constants.MINUTE_TO_SECONDS
-        candles = await self.exchange.get_symbol_prices(symbol, time_frame)
+
+        if self.start_timestamp is not None or self.end_timestamp is not None:
+            since = self.start_timestamp
+            if self.start_timestamp is None or self.start_timestamp < await self.get_first_candle_timestamp(symbol, time_frame):
+                since=0
+            candles = await self.exchange.get_symbol_prices(symbol, time_frame, since=since)
+            while since < candles[-1][0] if not self.end_timestamp \
```

Comment:
```suggestion
            while since < candles[-1][common_enums.PriceIndexes.IND_PRICE_TIME.value]] if not self.end_timestamp \
```
for each [0] which refers to the timestamp value
```python
import octobot_commons.enums as common_enums
```
(with this as import)